fast_api_als/routers/submit_lead.py

```diff
@@ -37,7 +39,7 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
     t1 = [int(time.time() * 1000.0)]
     
     if not db_helper_session.verify_api_key(apikey):
-        # throw proper fastpi.HTTPException
+        error("session is not verified")
```

Comment:
if the API key isn't verified, then you should throw an HTTPException, isn't it?